website/client/src/components/tasks/task.vue

```diff
@@ -395,22 +421,25 @@
     border-radius: 2px;
     position: relative;
 
-    &:hover:not(.task-not-editable) {
+    &:hover:not(.task-not-editable),
```

Comment:
@alexandreabeh it looks like these changes adding styles for `focus-visible` broke the `:hover` ones

Reply:
Ok, I'll take a look